configure.ac

```diff
@@ -55,7 +55,7 @@ compiler_options=""
 STRICT_PROTOTYPES=""
 echo "Detected compiler: $compiler"
 if test "$compiler" = "gcc" ; then
-   compiler_options="-Wstrict-prototypes -Wall -Werror -ansi -fstack-protector-all -Wstack-protector"
```

Comment:
It's OK for us. Only MinGW for Windows fails to build with "-ansi", it claims that strdup does not exist. We always do cross-compilation from Linux, so you can just skip this patch.